extra/redis/redis-tests.factor

```diff
@@ -66,3 +66,17 @@ IN: redis.tests
         [ "hello" redis-incr ] [ drop t ] recover
     ] with-redis-test
 ] unit-test
+
+{ "e0e1f9fabfc9d4800c877a703b823ac0578ff8db" } [
+    [ "return 1" redis-script-load ] with-redis-test
+] unit-test
+
+{ { 0 0 } } [
+    [ { "foo" "bar" } redis-script-exists ] with-redis-test
```

Comment:
should this return ``t/f`` instead of ``1/0``?

Reply:
It can be. But I don't see any other commands massaging the redis output. Is it worth doing it here?